build.gradle

```diff
@@ -1,14 +1,13 @@
 plugins {
     id "java-library"
     id 'signing'
-    id "com.github.hierynomus.license" version "0.15.0"
```

Comment:
Just comment it out instead of removing it? `code-quality.gradle` depends on this plugin.